### ISSUE TITLE
Rename chainscript

### ISSUE THREAD

**Issue by muursh** [mintlayer/building_on_mintlayer#4]

**Comment by iljakuklic** [comment]
Is this issue about coming up with a name for Mintlayer-flavoured bitcoin script dialect? As far as code base is concerned, all references to chainscript disappeared when the repo was moved away from Substrate as the crate is just called `script` now. That is not supposed to be a public facing name and neither was chainscript ever intended to be.

**Comment by muursh** [comment]
> Is this issue about coming up with a name for Mintlayer-flavoured bitcoin script dialect?

Yeah. Essentially. And it's more of a reminder than a real issue I just happened to have gh open when it came up
